tests/functional/vault/test_management.py

```diff
@@ -0,0 +1,72 @@
+import pytest
```

Comment:
Can you add these test additional checks to the other cases in the test suite instead?

Reply:
fixed!